### ISSUE TITLE
Please add scaladocs

### ISSUE THREAD

**Issue by nafg** [com-lihaoyi/sourcecode#148]
The readme file is great but when I'm working in my IDE I can't tell what things do without open my browser to github.

**Comment by jducoeur** [comment]
As with pretty much everything in the Scala ecosystem, [the Scaladocs can be found on javadoc.io](https://www.javadoc.io/doc/com.lihaoyi/sourcecode_2.13/latest/index.html).

**Comment by lihaoyi** [comment]
I think he means having actual _docs_ there, rather than just a list of function signatures

**Comment by nafg** [comment]
Right, my IDE can also show scaladocs without going to javadoc.io, but the information isn't there, it's only in the readme

**Comment by nafg** [comment]
One day soon we'll be able just tell ChatGPT to open PRs like this...
